db-scripts/src/main/resources/cgds.sql

```diff
@@ -865,5 +869,46 @@ CREATE TABLE `info` (
   `DB_SCHEMA_VERSION` varchar(24),
   `GENESET_VERSION` varchar(24)
 );
+
+-- --------------------------------------------------------
+CREATE TABLE `resource_definition` (
+  `RESOURCE_ID` varchar(255) NOT NULL,
+  `DISPLAY_NAME` varchar(255) NOT NULL,
+  `DESCRIPTION` varchar(2048) NOT NULL,
+  `RESOURCE_TYPE` varchar(255) NOT NULL COMMENT 'STUDY, PATIENT or SAMPLE',
```

Comment:
may be declare it as enum
`RESOURCE_TYPE ENUM('STUDY', 'PATIENT', 'SAMPLE') NOT NULL,`